packages/backend/axios-layer/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "axios-layer",
```

Comment:
Probably would be easier to just roll this into the `ethers-layer`, but this is ok.

Reply:
I think we should keep them separate, so we won't create fat layers.